CombinatorialGames/IGame/IGame.lean

```diff
@@ -15,32 +16,132 @@ set_option linter.dupNamespace false
 -- All computation should be done through `IGame.Short`.
 noncomputable section
 
-open PGame Set Pointwise
+-- This avoids name clashes with the existing `PGame`.
+-- Remove it when we finish porting!
```

Comment:
(could add a TODO here, though I don't suppose we'll forget about removing this anytime soon.)